scapy/plist.py

```diff
@@ -21,8 +22,10 @@
 from scapy.extlib import plt, MATPLOTLIB_INLINED, MATPLOTLIB_DEFAULT_PLOT_KARGS
 from functools import reduce
 import scapy.modules.six as six
+from matplotlib.lines import Line2D
```

Comment:
`matplotlib` is not mandatory. You must get `Line2D` from `scapy/extlib.py`  by importing it correctly, and setting it to `None` if the import fails.

Reply:
Thanks! Added that import. The code quality build tool unfortunately doesn't recognize imports used in typing. Not sure how to fix that

Reply:
Sorry, I should have provided an example. I meant that you should do this:
```
git diff
diff --git a/scapy/extlib.py b/scapy/extlib.py
index 91dec81d..7d8dd6aa 100644
--- a/scapy/extlib.py
+++ b/scapy/extlib.py
@@ -20,6 +20,7 @@ from scapy.error import log_loading
 try:
     from matplotlib import get_backend as matplotlib_get_backend
     from matplotlib import pyplot as plt
+    from matplotlib.lines import Line2D
     MATPLOTLIB = 1
     if "inline" in matplotlib_get_backend():
         MATPLOTLIB_INLINED = 1
@@ -29,6 +30,7 @@ try:
 # RuntimeError to catch gtk "Cannot open display" error
 except (ImportError, RuntimeError):
     plt = None
+    Line2D = None
     MATPLOTLIB = 0
     MATPLOTLIB_INLINED = 0
     MATPLOTLIB_DEFAULT_PLOT_KARGS = dict()
```

And then:
```
from scapy.extlib import plt, Lines2D, MATPLOTLIB_INLINED, \                    
    MATPLOTLIB_DEFAULT_PLOT_KARGS 
```

Reply:
After updating the extlib file as above, do I need to make changes to any test files? Had a look at regression.uts and can't quite figure out what needs to be updated if at all